.claude/skills/pr-review/SKILL.md

```
@@ -0,0 +1,58 @@
+
+### Layer 1 — deterministic scanners (the cheap, certain half)
+Run the same OSS tools CodeRabbit runs, locally, scoped to the diff:
+```bash
+make scan                 # from the repo root, or:
+bash scripts/pr-scan.sh   # optional first arg: a base ref (default: merge-base with origin/terraform)
+```
+It auto-selects tools by changed file type (`.tf` → terraform fmt + tflint + checkov + trivy; secrets → gitleaks; Dockerfile → hadolint; workflows → actionlint; shell/yaml/md → shellcheck/yamllint/markdownlint; SAST → semgrep). Checkov picks up the repo's `.checkov.yaml` automatically. Fix everything it reports — these are exactly the comments CodeRabbit's tool layer would post. It names any tool it couldn't run (not installed) so coverage gaps are explicit; widen coverage by installing them (see `docs/pr-review-gate.md`).
+
+### Layer 2 — LLM multi-lens adversarial review (the judgment half)
+The scanners can't reason about intent, design, or cross-module contracts. For that, run the engine via the Workflow tool:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)